sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.     invoice	text,
</pre><p>Use spaces instead of tabs in this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done, replaces all tabs to spaces</p>